cmd/syncEnumerator.go

```diff
@@ -96,7 +96,7 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 				atomic.AddUint64(&cca.atomicSourceFilesScanned, 1)
 			}
 		}, nil, cca.s2sPreserveBlobTags, cca.logVerbosity.ToPipelineLogLevel(), cca.cpkOptions, nil /* errorChannel */, objectIndexerMap, tqueue, true /* isSource */, true, /* isSync */
-		1 /* maxObjectIndexerSizeInGB */, time.Time{} /* lastSyncTime (not used by source traverser) */, cca.cfdMode)
+		cca.maxObjectIndexerMapSizeInGB /* maxObjectIndexerSizeInGB */, time.Time{} /* lastSyncTime (not used by source traverser) */, cca.cfdMode)
```

Comment:
nit, we don't need comment for reasonably named variables (only for literals)
so /* maxObjectIndexerSizeInGB */ goes away